proto/cmp/services/transport/v1alpha/trip_types.proto

```diff
@@ -84,41 +87,41 @@ message TripSegment {
   // flight: "Y", "J", "F".
   // train: "1st", "2nd" or "S", "C", "P"
   // transfer: "S", "P", "VIP", "SS"
-  string service_type_code = 8;
+  string service_type_code = 9;
 
   // Service type description examples:
   // flight: "Economy Class", "Business Class", "First Class"
   // train: "First Class", "Second Class" or "standard", "comfort", and "premium",..
   // transfer: "Shuttle", "Private", "VIP Limosine", "Speedy Shuttle",...
-  string service_type_description = 9;
+  string service_type_description = 10;
 
   // Trip Time
   //
   // Ex: Time(hours=2, minutes=45)
-  cmp.types.v1alpha.Time trip_time = 10;
+  cmp.types.v1alpha.Time trip_time = 11;
```

Comment:
using time but naming the field trip_duration would be enough, right? If we would create a type that is cmp.types.v1alpha.Duration, it would still have Time values... with the same two fields Hours and Minutes

Reply:
> it would still have Time values... with the same two fields Hours and Minutes

It can work. However, Google also has a [duration message type](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/duration.proto), it's a real "duration" object with seconds and nanos. 
In my opinion, we can have a simple duration message type with only minutes (no hours) like:

```protobuf
message Duration {
  int32 minutes = 1;
}
```

If we go with this option we should also update min/max duration fields in transport and activities `search_parameters_types.proto` files.

Reply:
Done